tests/gateway/providers/test_bedrock.py

```diff
@@ -0,0 +1,480 @@
+        "request": {"prompt": "How does a car work?", "max_tokens": 200},
+    },
+    {
+        "provider": AWSBedrockModelProvider.AMAZON,
+        "config": {
+            "name": "completions",
+            "route_type": "llm/v1/completions",
+            "model": {
+                "provider": "bedrock",
+                "name": "amazon.titan-tg1-large",
+            },
+        },
+        "request": {
+            "prompt": "This is a test",
+        },
+        "response": {
+            "results": [
+                {
+                    "tokenCount": 5,
+                    "outputText": "\nThis is a test",
+                    "completionReason": "FINISH",
+                }
+            ],
+            "inputTextTokenCount": 4,
+        },
+        "expected": {
+            "candidates": [{"metadata": {}, "text": "\nThis is a test"}],
+            "metadata": {
+                "model": "amazon.titan-tg1-large",
+                "route_type": "llm/v1/completions",
+                "input_tokens": None,
+                "output_tokens": None,
+                "total_tokens": None,
+            },
+        },
+    },
+    {
+        "provider": AWSBedrockModelProvider.AI21,
+        "config": {
+            "name": "completions",
+            "route_type": "llm/v1/completions",
+            "model": {
+                "provider": "bedrock",
+                "name": "ai21.j2-ultra",
+            },
+        },
+        "request": {
+            "prompt": "This is a test",
+        },
+        "response": ai21_completion_response(),
+        "expected": ai21_parsed_completion_response(),
+    },
+    {
+        "provider": AWSBedrockModelProvider.AI21,
+        "config": {
+            "name": "completions",
+            "route_type": "llm/v1/completions",
+            "model": {
+                "provider": "bedrock",
+                "name": "ai21.j2-mid",
+            },
+        },
+        "request": {
+            "prompt": "This is a test",
+        },
+        "response": ai21_completion_response(),
+        "expected": ai21_parsed_completion_response(),
+    },
+    {
+        "provider": AWSBedrockModelProvider.COHERE,
+        "config": {
+            "name": "completions",
+            "route_type": "llm/v1/completions",
+            "model": {
+                "provider": "bedrock",
+                "name": "cohere.command",
+            },
+        },
+        "request": {
+            "prompt": "This is a test",
+        },
+        "response": cohere_completions_response(),
+        "expected": {},
+    },
+]
+
+bedrock_live_aws_configs = [
+    {"aws_region": "us-east-1"},
+    pytest.param(
+        {
+            "aws_access_key_id": os.environ.get("AWS_ACCESS_KEY_ID"),
+            "aws_secret_access_key": os.environ.get("AWS_SECRET_ACCESS_KEY"),
+            "aws_session_token": os.environ.get("AWS_SESSION_TOKEN"),
+        },
+        marks=pytest.mark.skipif(
```

Comment:
Why do we need to skip these tests? 
Since all tests apart from the last test (which is an integration test and shouldn't be in this suite) are mocking portions of access, we should be using `monkeypatch.setenv()` for the `AWS_x` environment keys so that the patched calls can be evaluated.